indigo/indigo-extras/src/main/scala/indigoextras/subsystems/FPSCounter.scala

```diff
@@ -42,15 +42,21 @@ final case class FPSCounter(
   def update(context: SubSystemFrameContext, model: FPSCounterState): GlobalEvent => Outcome[FPSCounterState] = {
     case FrameTick =>
       if (context.gameTime.running >= (model.lastInterval + Seconds(1)))
```

Comment:
Since `targetFPS` value won't change, this could be a curried function outside of `update`.